.github/workflows/build_and_test.yml

```diff
@@ -72,14 +81,18 @@ jobs:
           java-version: 21
           distribution: 'adopt'
           cache: 'gradle'
-      - name: start minikube
+      - name: Set Up Minikube
+        uses: medyagh/setup-minikube@v0.0.18
```

Comment:
Could you make a separate PR for using this new GitHub Action? Some GitHub Action job fails after merging.

We can merge this part seperately, @TQJADE .

Reply:
@dongjoon-hyun https://github.com/apache/spark-kubernetes-operator/pull/82

Reply:
Thank you!